src/fparser/common/tests/test_readfortran.py

```diff
@@ -1730,3 +1730,22 @@ def test_conditional_include_omp_conditional_liness_free_format_multiple():
     line = reader.next()
     # 8 spaces in input_text, plus two for replacing the !$
     assert line.line == "bla          bla"
+
+
+def test_process_directives_option_read_fortran():
+    """Test handling of the process_directives option."""
```

Comment:
Pls add a note to say that the tests for the actual functionality are in two/tests/test_comments_and_directives.py

Reply:
Done.